packages/web-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,147 @@
+import { css } from '@microsoft/fast-element';
+import { forcedColorsStylesheetBehavior } from '@microsoft/fast-foundation';
+import {
+  accentFillActiveBehavior,
+  accentFillHoverBehavior,
+  accentFillRestBehavior,
+  neutralFillHoverBehavior,
+  neutralFillInputActiveBehavior,
+  neutralFillInputHoverBehavior,
+  neutralFillInputRestBehavior,
+  neutralFillRestBehavior,
+  neutralFocusBehavior,
+  neutralForegroundRestBehavior,
+  neutralOutlineRestBehavior,
+} from '../styles/index';
+
+export const TooltipStyles = css`
+  :host {
+    position: relative;
+    contain: layout;
+    overflow: visible;
+    height: 0;
+    width: 0;
+  }
+
+  .tooltip {
+    box-sizing: border-box;
+    border-radius: calc(var(--corner-radius) * 1px);
+    border: calc(var(--outline-width) * 1px) solid transparent;
+    background: ${neutralForegroundRestBehavior.var};
+    color: ${neutralFillRestBehavior.var};
+    padding: 4px 12px;
+    height: fit-content;
+    width: fit-content;
+    font-family: var(--body-font);
+    font-size: var(--type-ramp-base-font-size);
+    line-height: var(--type-ramp-base-line-height);
+    white-space: nowrap;
+    // TODO: a mechanism to manage z-index across components
+    // https://github.com/microsoft/fast/issues/3813
+    z-index: 10000;
+  }
+
+  fluent-anchored-region {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    overflow: visible;
+    flex-direction: row;
+  }
+
+  fluent-anchored-region.right,
+  fluent-anchored-region.left {
+    flex-direction: column;
+  }
+
+  fluent-anchored-region.top .tooltip::after {
+    content: '';
+    width: 12px;
+    height: 12px;
+    background: ${neutralForegroundRestBehavior.var};
+    border-radius: calc(var(--corner-radius) * 1px);
+    position: absolute;
+    transform: rotate(45deg) translateX(-50%);
+    bottom: 4px;
+    left: 50%;
+  }
+
+  fluent-anchored-region.top .tooltip {
+    margin-bottom: 12px;
+  }
+
+  fluent-anchored-region.bottom .tooltip::after {
+    content: '';
+    width: 12px;
+    height: 12px;
+    background: ${neutralForegroundRestBehavior.var};
+    border-radius: calc(var(--corner-radius) * 1px);
+    position: absolute;
+    transform: rotate(45deg) translateX(-50%);
+    top: 12px;
+    left: 50%;
+  }
+
+  fluent-anchored-region.bottom .tooltip {
+    margin-top: 12px;
+  }
+
+  fluent-anchored-region.left .tooltip::after {
+    content: '';
+    width: 12px;
+    height: 12px;
+    background: ${neutralForegroundRestBehavior.var};
+    border-radius: calc(var(--corner-radius) * 1px);
+    position: absolute;
+    transform: rotate(45deg) translateY(-50%);
+    top: 50%;
+    right: 12px;
+  }
+
+  fluent-anchored-region.left .tooltip {
+    margin-right: 12px;
+  }
```

Comment:
It looks like these two blocks are mistaken copies of lines 57-71.

Reply:
the majority of the properties are the same. the difference is 51-71 is positioned absolute, at the top of the anchor

Reply:
@khamudom we shouldn't need to redeclare *every* CSS property for each style block though. Please combine selectors where we can.

Reply:
updated with combined selectors